packages/contracts/contracts/token/Vesting.sol

```diff
@@ -182,14 +179,25 @@ contract Vesting is IVesting, AccessControl, ReentrancyGuard {
             vesting.amount == 0 || vesting.cliffMonths == cliffMonths,
             "vesting already exists with different cliff"
         );
+        require(
+            vesting.amount == 0 || vesting.vestingMonths == vestingMonths,
+            "vesting already exists with different vesting period"
+        );
 
         vesting.cliffMonths = cliffMonths;
-        vesting.vestingMonths = PRIVATE_SALE_VESTING_MONTHS;
+        vesting.vestingMonths = vestingMonths;
         vesting.amount += amount;
 
         totalPrivateSales += amount;
     }
 
+    function setStartTime(uint256 _startTime)
+        public
+        onlyRole(DEFAULT_ADMIN_ROLE)
```

Comment:
can we add a new role for this? Or maybe rename the `PRIVATE_SELLER` to something that accomodates the two? it's usually better to have that than to restrict this to the root

Reply:
Can't think of a better name. What were you thinking?